bin/src/display/generic_framebuffer.rs

```diff
@@ -73,48 +68,6 @@ pub trait GenericFramebuffer: Send + 'static {
         buffer: &[(u8, u8, u8)], // rgb, row-wise, left-to-right, top-to-bottom
     );
 
```

Comment:
But I really like orca mode 😭 As I understand the binary size is really important for the TP-Link, yes? Would it be possible to put this behind a compile flag for just the orbic so that we can keep this feature while keeping the tp-link binary smaller?